src/index.html

```diff
@@ -31,8 +32,9 @@ <h2>Available Shows</h2>
     <div class="screen hidden"></div>
 
     <footer>
-        Created by <a href="https://github.com/Peter1907">Peter Beshara</a>&
+        Created by <a href="https://github.com/Peter1907">Peter Beshara </a>&
         <a href="https://github.com/DhuruvSingh">Dhuruv Singh</a>
     </footer>
+    <script src="./index.js" type="modules"></script>
```

Comment:
Also you don't have to include a script tag at the end

Reply:
ok ill remove it

Reply:
ok ill remove it